test/test_sparse.py

```diff
@@ -3575,6 +3576,12 @@ def test_future_empty_dim(self, device, dtype, op):
 
             t = sample_input.input
             mask = sample_input_kwargs.get('mask')
+            if mask is None and op_name in {'prod', 'amax', 'amin'}:
+                # FIXME: for now reductions with non-zero reduction identity and
+                # unspecified mask are not supported for sparse COO
+                # tensors, see torch._masked.prod implementation
```

Comment:
But doesn't this PR introduce that exact support?

Reply:
When the mask is *unspecified* (for CSR prod at least) we're raising a ValueError (see the comment [here](https://github.com/pytorch/pytorch/blob/master/torch/_masked/__init__.py#L761-L772)). I believe the same logic applies to COO for cases with nonzero reduction identities so I followed this convention. Does that make sense?